packages/meditrak-server/src/database/models/Question.js

```diff
@@ -35,14 +35,19 @@ export class QuestionModel extends DatabaseModel {
   }
 }
 
-const onChangeUpdateDataElement = async ({ type: changeType, record }, models) => {
-  const { code, data_source_id: dataSourceId } = record;
-
+const onChangeUpdateDataElement = async (
+  { type: changeType, old_record: oldRecord, new_record: newRecord },
+  models,
+) => {
   switch (changeType) {
-    case 'update':
+    case 'update': {
+      const { code, data_source_id: dataSourceId } = newRecord;
```

Comment:
Same question as earlier - should we delete the old data source, if the data source id doesn't match the old record?

Reply:
Ok :+1: